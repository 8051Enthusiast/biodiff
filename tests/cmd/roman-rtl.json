{
  "version": "1",
  "presets": {
    "global": [
      {
        "name": "default",
        "gap_open": -5,
        "gap_extend": -1,
        "mismatch_score": -1,
        "match_score": 0,
        "mode": {
          "Blockwise": 8192
        },
        "backend": {
          "backend": "wfa2"
        }
      },
      {
        "name": "global",
        "gap_open": -5,
        "gap_extend": -1,
        "mismatch_score": -1,
        "match_score": 0,
        "mode": "Global",
        "backend": {
          "backend": "wfa2"
        }
      }
    ],
    "current_global": 0,
    "semiglobal": [
      {
        "name": "default",
        "gap_open": -5,
        "gap_extend": -1,
        "mismatch_score": -1,
        "match_score": 0,
        "mode": "Semiglobal",
        "backend": {
          "backend": "wfa2"
        }
      }
    ],
    "current_semiglobal": 0
  },
  "style": {
    "mode": "Roman",
    "ascii_col": false,
    "bars_col": false,
    "layout": "Horizontal",
    "spacer": false,
    "right_to_left": true,
    "column_count": "Fit",
    "no_scroll": false
  }
}