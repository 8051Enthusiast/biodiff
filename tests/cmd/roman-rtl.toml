bin.name = "biodiff"
args = ["--color", "--columns=4", "--config", "tests/cmd/roman-rtl.json", "--print", "-gglobal", "tests/text_x.txt", "tests/text_y.txt"]
status.code = 0
