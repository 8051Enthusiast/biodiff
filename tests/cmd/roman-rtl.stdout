[38;5;8m [39m      CXV        CV       CIV    LXXXIV [38;5;8m0000  | [39m      CXV        CV       CIV    LXXXIV [38;5;8m0000 [39m
[38;5;8m [39m    XXXII       CXV        CV     XXXII [38;5;8m0004  | [39m    XXXII       CXV        CV     XXXII [38;5;8m0004 [39m
[38;5;8m [39m       CI       CIX       CXI       CXV [38;5;8m0008  | [39m       CI       CIX       CXI       CXV [38;5;8m0008 [39m
[38;5;8m [39m      CIX       CXI      XCIX     XXXII [38;5;8m000c  | [39m      CIX       CXI      XCIX     XXXII [38;5;8m000c [39m
[38;5;8m [39m    XXXII        CX       CXI       CIX [38;5;8m0010  | [39m    XXXII        CX       CXI       CIX [38;5;8m0010 [39m
[38;5;8m [39m     CXVI       CXX        CI      CXVI [38;5;8m0014  | [39m     CXVI       CXX        CI      CXVI [38;5;8m0014 [39m
[38;5;8m [39m     CXVI        CI    XCVIII     XXXII [38;5;8m0018  | [39m     CXVI        CI    XCVIII     XXXII [38;5;8m0018 [39m
[38;5;8m [39m       CX        CI        CI      CXIX [38;5;8m001c  | [39m       CX        CI        CI      CXIX [38;5;8m001c [39m
[38;5;8m [39m     CXVI       CXI    XCVIII     XXXII [38;5;8m0020  | [39m     CXVI       CXI    XCVIII     XXXII [38;5;8m0020 [39m
[38;5;8m [39m       CV       CII     XXXII       CIV [38;5;8m0024  | [39m       CV       CII     XXXII       CIV [38;5;8m0024 [39m
[38;5;8m [39m     XLVI       CXV        CI     CVIII [38;5;8m0028  | [39m     XLVI       CXV        CI     CVIII [38;5;8m0028 [39m
[38;5;8m [38;5;10m       CV       CIV    LXXXIV     XXXII [38;5;8m002c  | [38;5;10m                                        [38;5;8m002c [39m
[38;5;8m [38;5;10m       CI      CXVI     XXXII       CXV [38;5;8m0030  | [38;5;10m                                        [38;5;8m002c [39m
[38;5;8m [38;5;10m       CV     XXXII      CXVI       CXX [38;5;8m0034  | [38;5;10m                                        [38;5;8m002c [39m
[38;5;8m [38;5;10m       CX       CXI     XXXII       CXV [38;5;8m0038  | [38;5;10m                                        [38;5;8m002c [39m
[38;5;8m [38;5;10m       CV     XXXII      CXXI     CVIII [38;5;8m003c  | [38;5;10m                                        [38;5;8m002c [39m
[38;5;8m [38;5;10m      CIV      CXVI     XXXII        CX [38;5;8m0040  | [38;5;10m                                        [38;5;8m002c [39m
[38;5;8m [38;5;10m      CII     XXXII       CXV        CV [38;5;8m0044  | [38;5;10m                                        [38;5;8m002c [39m
[38;5;8m [38;5;10m     CXVI       CXV      CXIV        CV [38;5;8m0048  | [38;5;10m                                        [38;5;8m002c [39m
[38;5;8m [38;5;10m    CVIII        CV       CII     XXXII [38;5;8m004c  | [38;5;10m                                        [38;5;8m002c [39m
[38;5;8m [39m   LXXXIV         X [38;5;10m     XLVI        CI [38;5;8m0050  | [39m   LXXXIV         X [38;5;10m                    [38;5;8m002c [39m
[38;5;8m [39m     XCIX     XXXII        CI       CIV [38;5;8m0054  | [39m     XCIX     XXXII        CI       CIV [38;5;8m002e [39m
[38;5;8m [39m    XCVII      CXIV     XCVII       CIV [38;5;8m0058  | [39m    XCVII      CXIV     XCVII       CIV [38;5;8m0032 [39m
[38;5;8m [39m     CXIV        CI      CXVI      XCIX [38;5;8m005c  | [39m     CXIV        CI      CXVI      XCIX [38;5;8m0036 [39m
[38;5;8m [39m    XXXII       CXV        CV     XXXII [38;5;8m0060  | [39m    XXXII       CXV        CV     XXXII [38;5;8m003a [39m
[38;5;8m [39m       CX        CV     XXXII [38;5;9m LXXXVIII [38;5;8m0064  | [39m       CX        CV     XXXII [38;5;9m   LXXXIX [38;5;8m003e [39m
[38;5;8m [39m    XCVII        CI      CXVI       CXV [38;5;8m0068  | [39m    XCVII        CI      CXVI       CXV [38;5;8m0042 [39m
[38;5;8m [39m      CII       CXI     XXXII         C [38;5;8m006c  | [39m      CII       CXI     XXXII         C [38;5;8m0046 [39m
[38;5;8m [39m        X      XLVI [38;5;9m   LXXXIX [39m    XXXII [38;5;8m0070  | [39m        X      XLVI [38;5;9m LXXXVIII [39m    XXXII [38;5;8m004a [39m
[38;5;8m [39m       CI       CIX       CXI   LXXXIII [38;5;8m0074  | [39m       CI       CIX       CXI   LXXXIII [38;5;8m004e [39m
[38;5;8m [39m     CXIV       CXI       CIX     XXXII [38;5;8m0078  | [39m     CXIV       CXI       CIX     XXXII [38;5;8m0052 [39m
[38;5;8m [39m       CI      CXVI     XXXII        CI [38;5;8m007c  | [39m       CI      CXVI     XXXII        CI [38;5;8m0056 [39m
[38;5;8m [39m     CXVI     XXXII      CXVI       CXX [38;5;8m0080  | [39m     CXVI     XXXII      CXVI       CXX [38;5;8m005a [39m
[38;5;8m [39m    XCVII       CIX     XXXII       CXI [38;5;8m0084  | [39m    XCVII       CIX     XXXII       CXI [38;5;8m005e [39m
[38;5;8m [39m      CXV     XXXII        CI      CVII [38;5;8m0088  | [39m      CXV     XXXII        CI      CVII [38;5;8m0062 [39m
[38;5;8m [39m    XXXII        CI      CXIV     CXVII [38;5;8m008c  | [39m    XXXII        CI      CXIV     CXVII [38;5;8m0066 [39m
[38;5;8m [39m    XXXII        CI       CIV      CXVI [38;5;8m0090  | [39m    XXXII        CI       CIV      CXVI [38;5;8m006a [39m
[38;5;8m [39m   XCVIII       CIX     CXVII        CX [38;5;8m0094  | [39m   XCVIII       CIX     CXVII        CX [38;5;8m006e [39m
[38;5;8m [39m      CXI     XXXII      CXIV        CI [38;5;8m0098  | [39m      CXI     XXXII      CXIV        CI [38;5;8m0072 [39m
[38;5;8m [39m    XCVII       CXV     XXXII       CII [38;5;8m009c  | [39m    XCVII       CXV     XXXII       CII [38;5;8m0076 [39m
[38;5;8m [39m     XCIX     XXXII        CI       CIX [38;5;8m00a0  | [39m     XCIX     XXXII        CI       CIX [38;5;8m007a [39m
[38;5;8m [39m    XCVII      CXIV     XCVII       CIV [38;5;8m00a4  | [39m    XCVII      CXIV     XCVII       CIV [38;5;8m007e [39m
[38;5;8m [39m     CXIV        CI      CXVI      XCIX [38;5;8m00a8  | [39m     CXIV        CI      CXVI      XCIX [38;5;8m0082 [39m
[38;5;8m [39m       CI    XCVIII     XXXII       CXV [38;5;8m00ac  | [39m       CI    XCVIII     XXXII       CXV [38;5;8m0086 [39m
[38;5;8m [39m       CI        CI      CXIX      CXVI [38;5;8m00b0  | [39m       CI        CI      CXIX      CXVI [38;5;8m008a [39m
[38;5;8m [39m      CIV      CXVI     XXXII        CX [38;5;8m00b4  | [39m      CIV      CXVI     XXXII        CX [38;5;8m008e [39m
[38;5;8m [39m       CI      CXVI     XXXII        CI [38;5;8m00b8  | [39m       CI      CXVI     XXXII        CI [38;5;8m0092 [39m
[38;5;8m [39m    XXXII       CXV      CXVI       CXX [38;5;8m00bc  | [39m    XXXII       CXV      CXVI       CXX [38;5;8m0096 [39m
[38;5;8m [39m      CXV     XXXII       CXV        CV [38;5;8m00c0  | [39m      CXV     XXXII       CXV        CV [38;5;8m009a [39m
[38;5;8m [39m       CV       CII       CII     CXVII [38;5;8m00c4  | [39m       CV       CII       CII     CXVII [38;5;8m009e [39m
[38;5;8m [39m       CX        CI        CV      XCIX [38;5;8m00c8  | [39m       CX        CI        CV      XCIX [38;5;8m00a2 [39m
[38;5;8m [39m    XXXII      CXXI     CVIII      CXVI [38;5;8m00cc  | [39m    XXXII      CXXI     CVIII      CXVI [38;5;8m00a6 [39m
[38;5;8m [39m      CIV      CIII        CV       CIV [38;5;8m00d0  | [39m      CIV      CIII        CV       CIV [38;5;8m00aa [39m
[38;5;8m [38;5;10m                    [39m        X      XLVI [38;5;8m00d4  | [38;5;10m      CIV    LXXXIV [39m        X      XLVI [38;5;8m00ae [39m
[38;5;8m [38;5;10m                                        [38;5;8m00d6  | [38;5;10m     CXVI     XXXII       CXV        CV [38;5;8m00b2 [39m
[38;5;8m [38;5;10m                                        [38;5;8m00d6  | [38;5;10m    XXXII      CXVI       CXX        CI [38;5;8m00b6 [39m
[38;5;8m [38;5;10m                                        [38;5;8m00d6  | [38;5;10m      CXI     XXXII       CXV        CV [38;5;8m00ba [39m
[38;5;8m [38;5;10m                                        [38;5;8m00d6  | [38;5;10m    XXXII      CXXI     CVIII        CX [38;5;8m00be [39m
[38;5;8m [38;5;10m                                        [38;5;8m00d6  | [38;5;10m     CXVI     XXXII        CX        CV [38;5;8m00c2 [39m
[38;5;8m [38;5;10m                                        [38;5;8m00d6  | [38;5;10m    XXXII       CXV        CV       CIV [38;5;8m00c6 [39m
[38;5;8m [38;5;10m                                        [38;5;8m00d6  | [38;5;10m      CXI      XCIX        CI       CXV [38;5;8m00ca [39m
[38;5;8m [38;5;10m                                        [38;5;8m00d6  | [38;5;10m      CII     XXXII         C        CX [38;5;8m00ce [39m
[38;5;8m [38;5;10m                                        [38;5;8m00d6  | [38;5;10m     XLVI        CI     CVIII        CV [38;5;8m00d2 [39m
[38;5;8m [39m     CXIV        CI     LXXII [38;5;10m          [38;5;8m00d6  | [39m     CXIV        CI     LXXII [38;5;10m    XXXII [38;5;8m00d6 [39m
[38;5;8m [39m      CXV        CV     XXXII        CI [38;5;8m00d9  | [39m      CXV        CV     XXXII        CI [38;5;8m00da [39m
[38;5;8m [39m      CIX       CXI       CXV     XXXII [38;5;8m00dd  | [39m      CIX       CXI       CXV     XXXII [38;5;8m00de [39m
[38;5;8m [39m      CXI       CIX     XXXII        CI [38;5;8m00e1  | [39m      CXI       CIX     XXXII        CI [38;5;8m00e2 [39m
[38;5;8m [39m     XCIX     XXXII        CI      CXIV [38;5;8m00e5  | [39m     XCIX     XXXII        CI      CXIV [38;5;8m00e6 [39m
[38;5;8m [39m      CXI       CIX       CIX       CXI [38;5;8m00e9  | [39m      CXI       CIX       CIX       CXI [38;5;8m00ea [39m
[38;5;8m [39m       CI      CXVI     XXXII        CX [38;5;8m00ed  | [39m       CI      CXVI     XXXII        CX [38;5;8m00ee [39m
[38;5;8m [39m     CXVI     XXXII      CXVI       CXX [38;5;8m00f1  | [39m     CXVI     XXXII      CXVI       CXX [38;5;8m00f2 [39m
[38;5;8m [39m     CXIV      CXIX     XXXII       CXI [38;5;8m00f5  | [39m     CXIV      CXIX     XXXII       CXI [38;5;8m00f6 [39m
[38;5;8m [39m       CV     XXXII      CXII     XCVII [38;5;8m00f9  | [39m       CV     XXXII      CXII     XCVII [38;5;8m00fa [39m
[38;5;8m [39m     CXII     CXVII     XXXII      CXVI [38;5;8m00fd  | [39m     CXII     CXVII     XXXII      CXVI [38;5;8m00fe [39m
[38;5;8m [38;5;10m                              [39m     XLVI [38;5;8m0101  | [38;5;10m                              [39m     XLVI [38;5;8m0102 [39m
